crates/uv-resolver/src/lock.rs

```diff
@@ -1160,24 +1160,24 @@ source = "registry+https://pypi.org/simple"
 
 [[distribution.wheel]]
 url = "https://files.pythonhosted.org/packages/14/fd/2f20c40b45e4fb4324834aea24bd4afdf1143390242c0b33774da0e2e34f/anyio-4.3.0-py3-none-any.whl"
+hash = "sha256:048e05d0f6caeed70d731f3db756d35dcc1f35747c8c403364a8332c630441b8"
 "#;
         let result: Result<Lock, _> = toml::from_str(data);
         insta::assert_debug_snapshot!(result);
     }
 
     #[test]
-    fn hash_optional_missing() {
+    fn hash_required_absent() {
         let data = r#"
 version = 1
 
 [[distribution]]
 name = "anyio"
 version = "4.3.0"
-source = "path+file:///foo/bar"
+source = "registry+https://pypi.org/simple"
 
 [[distribution.wheel]]
-url = "file:///foo/bar/anyio-4.3.0-py3-none-any.whl"
-hash = "sha256:048e05d0f6caeed70d731f3db756d35dcc1f35747c8c403364a8332c630441b8"
```

Comment:
Hmmm the purpose of this test was to check a case where a hash is given _but should not be present_. And the purpose of the test above was to check a case where a hash is missing _but should be present_. i.e., They should both be checking error cases.